opentasks-provider/src/main/java/org/dmfs/provider/tasks/processors/instances/Validating.java

```diff
@@ -43,50 +69,79 @@ public Validating(EntityProcessor<InstanceAdapter> delegate)
     public InstanceAdapter insert(SQLiteDatabase db, InstanceAdapter entityAdapter, boolean isSyncAdapter)
```

Comment:
Could this method be refactored to something like this?:
```java
    @Override
    public InstanceAdapter insert(SQLiteDatabase db, InstanceAdapter entityAdapter, boolean isSyncAdapter)
    {
        validateNotSyncadapter(isSyncAdapter);
        validateOnlyAllowedFieldsAreUpdated(entityAdapter);
        validateInstanceDoesNotExistAlready(db, entityAdapter);
        
        return mDelegate.insert(db, entityAdapter, false);
    }
```
I think it could be clearer and more self-documenting.

Reply:
updated

Reply:
It's nice, I think